### ISSUE TITLE
Is This Lib Dead??

### ISSUE THREAD

**Issue by Nagavikas96** [aksonov/react-native-router-flux#3817]
Hii router flux maintainers,

We are using React-native-router-flux since last 3.5 years, But since last year there is no update in lib, there is a update in react navigation v5 and v6, and many react native version upgrades, Can we expect next version from React Native Router flux?, 

If this lib not going to maintain, I need to prepare my whole project to use react-navigation, please let us know...

**Comment by phatmovista** [comment]
I had the same questions and already assumed it is dead, but on the bright side, the latest version of React Navigation gives you very similar router config syntax so making the switch is not very painful if you can give it a try

**Comment by vijaychouhan-rails** [comment]
Yes, this is dead and should move to react-navigation, I moved to react-navigation latest version

**Comment by tanz98** [comment]
is this error caused because this library?

![image](https://user-images.githubusercontent.com/35478469/178889167-d1cbbcca-0d5c-4fdf-87d3-f9cf80876e0c.png)

i open the old project in my new mac m1, i don't know what happen my project when run in release architecture got this error. but when i try build from scratch & using router flux i always got this in release architecture when running the project

**Comment by varemenos** [comment]
@phatmovista

can you point to the said API/documentation?

**Comment by nica0012** [comment]
:(
